.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: node_js
 node_js:
-  - "0.8"
-  - "0.10"
```

Comment:
Please add these back; this version of the module supports these Node.js versions. Removing support is a major version bump. You can retarget your PR to the 2.0 branch if you want to make a breaking change, though.

Reply:
I'll retarget as typescript won't work with such old versions of node. 

I'll also thinking that this maybe better pushed to @DefinitelyTyped. This way there would be locked @typings/router@1.3.3 version aligned with a publish router@1.3.3 and leave it to the community to update. I often use older package versions simply due to the @type support. 

Looking down the list of PRs for 2.0 I not sure what methods we could do to keep these aligned, except by being rigorous for any forthcoming changes. 

Up to you guys really. Let me know.

Reply:
Looks like I can roll typescript back to 1.8.0 and it should work with node >= 0.8.0